libs/runtime/unit/platform_unit_test.cpp

```diff
@@ -1094,7 +1094,7 @@ TEST_CASE("get_authentication_id", "[platform]")
 
 #define SEQUENCE_LENGTH 1024 * 128
 #define CHI_SQUARED_STATISTIC_THRESHOLD \
-    3.841 // Critical value for Chi-squared test with 2 degrees of freedom with significance level of 0.05.
+    9.210 // Critical value for Chi-squared test with 2 degrees of freedom with significance level of 0.01.
```

Comment:
For my understanding, how does the change in level help?
Was there a test failure/problem already encountered?

Reply:
Yes, see #2986

Reply:
The issue doesn't provide the Root Cause Analysis.